README.md

```diff
@@ -76,6 +76,14 @@ You'll probably want to run the tests; here's how:
 Note the use of ramdisk as `TMPDIR`: some tests create temporary files, which
 slows them down if `TMPDIR` is on HDD or even SSD.
 
+We check the formatting of the Rust code during CI using
+[rust-fmt](https://github.com/rust-lang/rustfmt).  To make sure your code is
+properly formated install and run rust-fmt:
+```
+rustup component add rustfmt
+cargo fmt
```

Comment:
I just noticed that above we sometimes use `$` to prefix shell command examples. To you prefer to start them with `$`?

Reply:
Ah and I also noticed that you seem to prefer tab indented blocks instead of ` ``` ` blocks, will change that as well.